docker/kafka-python/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+docker build -t kafka-python .
+docker tag kafka-python us.gcr.io/broad-dsp-gcr-public/monster-kafka-python
```

Comment:
We should use a real version number in case we end up pushing more stuffs into the image.

Can you `declare -r` a version at the beginning for us to use, and append it to the commands that need it?